storage/storage.go

```diff
@@ -2349,6 +2349,7 @@ func toProtoChecksums(sendCRC32C bool, attrs *ObjectAttrs) *storagepb.ObjectChec
 }
```

Comment:
Can you also update the docs for NewGRPCClient and the summary for gRPC in doc.go to indicate which methods are not supported?